pnpm-workspace.yaml

```diff
@@ -2,6 +2,8 @@ packages:
   # documentation sites
   - 'apps/*'
   # npm publishable packages
-  - 'packages/**/*'
+  - 'packages/*'
+  - 'packages/dialtone-icons/*'
+  - 'packages/language-server/*'
```

Comment:
Wouldn't the previous `'packages/**/*'` cover this?

Reply:
That was too general, I was seeing unnamed packages that I didn't knew where they were coming from, so this way is more controlled.